melodic/distribution.yaml

```diff
@@ -946,6 +946,19 @@ repositories:
       url: https://github.com/osrf/capabilities.git
       version: master
     status: maintained
+  carViz:
```

Comment:
This is an unusual name for the key.  I'd suggest making this the same as the package, namely "jderobot_carviz".

Reply:
Solved in a new PR [#25551 ](https://github.com/ros/rosdistro/pull/25551)

Reply:
Please, if you open a new PR, close the old one.  That way we don't waste time looking at outdated stuff.